compiler/src/dotty/tools/dotc/core/TypeComparer.scala

```diff
@@ -1768,7 +1768,7 @@ class TypeComparer(@constructorOnly initctx: Context) extends ConstraintHandling
    *  Normalization is as follows: If `tp2` contains a skolem to its refinement type,
    *  rebase both itself and the member info of `tp` on a freshly created skolem type.
    */
-  protected def hasMatchingMember(name: Name, tp1: Type, tp2: RefinedType): Boolean =
+  def hasMatchingMember(name: Name, tp1: Type, tp2: RefinedType): Boolean =
```

Comment:
The comment to `hasMatchingMember` has a typo. It should be "of the refinement >>of<< `tp2`?".